migration_scripts/upgrade_from_2.4_to_2.5.py

```diff
@@ -0,0 +1,30 @@
+"""
+Simply makes UsefulLife an int.
+"""
+import argparse
+from lxml import etree as ET
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("file_path", type=str, help="path to file to update")
+    return parser.parse_args()
+
+def update_usefulLife(tree):
+    for usefulLife in tree.findall('.//*{http://buildingsync.net/schemas/bedes-auc/2019}UsefulLife'):
+        usefulLife.text = usefulLife.text.split(".", 1)[0]
+
+def main():
+    file_path = get_args().file_path
+
+    try:
+        tree = ET.parse(file_path)
+    except (FileNotFoundError, ET.ParseError) as e:
+        exit(f"File could not be read \n{e} \naborting...")
+
+    update_usefulLife(tree)
+    
+    tree.write(file_path)
```

Comment:
Can you save the original file under a different name like `ye_olde_excemelle` or a sober and more appropriate suffix like `my_xml_old`?